.pipelines/templates/template-rp-and-billing-e2e.yml

```diff
@@ -0,0 +1,78 @@
+# Azure DevOps Pipeline running RP e2e and Billing e2e
```

Comment:
there's a lot of duplication with e2e-with-billing.yml - can this be removed?

Reply:
feels like e2e-with-billing should be able to call this